docs/xshareclasses.md

```diff
@@ -153,6 +153,12 @@ When you specify `-Xshareclasses` without any parameters and without specifying
 
 The option `enableBCI` is enabled by default. However, if you use the `cacheRetransformed` option, this option forces cache creation into `-Xshareclasses:disableBCI` mode.
 
+### `checkURLTimestamps`
+
+        -Xshareclasses:checkURLTimestamps
+
+: Causes timestamps of `jar` or `zip` files to be checked every time a class is loaded. If a timestamp has changed, the class is loaded from the `jar` or `zip` file and not from the shared cache. This sub-option is not enabled by default and reflects the legacy behavior of the shared classes cache. This sub-option does not affect the behavior of bootstrap `jar` or `zip` files, which check timestamps only when `jar` or `zip` files are added to a class loader and are used for the first time to look for a class (same behavior as [`noCheckURLTimestamps`](xshareclasses.md#nocheckurltimestamps)).
```

Comment:
> "which check timestamps only when `jar` or `zip` files are added to a class loader and are used for the first time to look for a class" 

This statement is inaccurate for bootstrap jar/zip file. The timestamp is checked once when the bootstrap jar/file file is opened and is used for the first time to load a class. Probably we do not need to include this implementation detail here. Might be fine to simply say "the timestamp of a bootstrap jar/zip file is checked once when it is used for the first time to load a class".

Reply:
Fixed